packages/language-server/src/server.ts

```diff
@@ -63,8 +64,10 @@ export function startLanguageServer(connection: vscode.Connection) {
 
 		// We don't currently support running the TypeScript and Astro plugin in the browser
 		if (params.initializationOptions.environment !== 'browser') {
-			pluginHost.registerPlugin(new AstroPlugin(documentManager, configManager, workspaceUris));
-			pluginHost.registerPlugin(new TypeScriptPlugin(documentManager, configManager, workspaceUris));
+			const languageServiceManager = new LanguageServiceManager(documentManager, workspaceUris, configManager);
+
+			pluginHost.registerPlugin(new AstroPlugin(languageServiceManager, configManager));
+			pluginHost.registerPlugin(new TypeScriptPlugin(languageServiceManager, configManager));
```

Comment:
Unrelated to the script changes. Previously we created a new LanguageServiceManager for each plugin, this is unnecessary as they can definitely share one. 

This should improve memory consumption and performance